temboardagent/spc.py

```diff
@@ -694,10 +694,10 @@ def _check_message_error(self, message):
             msg = 'PostgreSQL backend error.'
             if message[1] is not None:
                 err_string = message[1]['string']
-                err_split = err_string.split('\x00')
-                msg = err_split[2][1:].decode()
+                err_split = err_string.split(b'\x00')
```

Comment:
Is it working with python 2 ?

Reply:
It seems like it doesn't fail with python2:

```
$ python2
>>> message = b'ERROR\x00VERROR\x00C42P01\x00Mthe message'
>>> message.split(b'\x00')
['ERROR', 'VERROR', 'C42P01', 'Mthe message']
>>> message = 'ERROR\x00VERROR\x00C42P01\x00Mthe message'
>>> message.split(b'\x00')
['ERROR', 'VERROR', 'C42P01', 'Mthe message']
```